src/com/mindscriptact/mvcExpressLogger/MvcExpressLogger.as

```diff
@@ -1,596 +1,637 @@
-package com.mindscriptact.mvcExpressLogger {
```

Comment:
this file is reformatted.  Brackets should stay on the line, and there is no need to indent package level.